### PR TITLE
[1] Migration to remove Sites

### PR DESCRIPTION
**Pull request by JR-G**
## Context

Once all the functionality is in place and has been syncing correctly we will then need to transition to the new model

## Link to Trello card

[](https://trello.com/c/yzG11Oym/183-site-uuids-transition-2-2-remove-the-old-site-model-spike)

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Review by cwrw — COMMENTED** [review]
can we remove any mention in the analytics files and tell the data team please

File: db/schema.rb [hunk 1]
```diff
@@ -818,7 +796,6 @@
   add_foreign_key "application_work_history_breaks", "application_forms", on_delete: :cascade
   add_foreign_key "candidates", "fraud_matches"
   add_foreign_key "course_options", "courses", on_delete: :cascade
-  add_foreign_key "course_options", "sites", on_delete: :cascade
```
Comment (cwrw) [inline db/schema.rb:821]: mental note to add this to temp sites when we add all the references etc.

**Review by carlosmartinez — APPROVED** [review]

**Comment by JR-G** [comment]
[Site UUIDS: Transition [2/2] - Remove the old site model [SPIKE]](https://trello.com/c/yzG11Oym/183-site-uuids-transition-2-2-remove-the-old-site-model-spike)

**Review by cwrw — APPROVED** [review]

**Pull request merged**